src/ui/FormWidgets/Checkbox.ts

```diff
@@ -71,12 +71,13 @@ export class Checkbox implements IFormWidgetWithLabel, IFormWidgetSelectable {
   }
 
   /**
-   * Checks the checkbox.
+   * Select the checkbox
+   * @param triggerChange will trigger change even if specified and not already selected
    */
-  public select() {
+  public select(triggerChange = true) {
```

Comment:
`select(true)` and `select(false)` is not as meaningful as something like `select()` and `selectWithoutChange()`. Just consider the option, you can leave it as is :)